lib/src/models/redditor.dart

```diff
@@ -30,7 +30,8 @@ class Redditor extends RedditBase
 
   Redditor.parse(Reddit reddit, Map data) : super.loadData(reddit, data) {
     if (!data.containsKey('name') &&
-        !(data.containsKey('kind') && data['kind'] == 't2')) {
+        !(data.containsKey('kind') &&
+            data['kind'] == Reddit.defaultRedditorKind)) {
```

Comment:
This indentation is weird, and would probably be better if it was aligned with the line above.

Reply:
You said dartfmt did this? Huh, okay, that's fine then. Just looked weird to me :)